inspirehep/records/api/base.py

```diff
@@ -250,12 +275,28 @@ def create(cls, data, **kwargs):
             record = super().create(data, id_=id_, **kwargs)
         return record
 
-    @classmethod
-    def get_linked_records_in_field(cls, data, path):
+    def get_records_pid_from_field(self, path):
+        """Returns list of tuples (pid_type, pid_value) of all linked records in field
```

Comment:
`Return a list of (pid_type, pid_value) tuples for all records referenced in the field at the given path`.

I know it's written in indicative mood in the whole file, but the first line of the docstring should be in imperative mood to conform with PEP 257 (I get really annoyed by these small things :smile_cat: ).